From c7b969d1a2a6b61bd179214ee2516b7b6cd55b27 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Tue, 9 Aug 2022 11:21:57 -0700
Subject: [PATCH 2/5] mbedtls: Remove unused variable

Fixes
library/bignum.c:1395:29: error: variable 't' set but not used [-Werror,-Wunused-but-set-variable]

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 lib/mbedtls-2.28.0/library/bignum.c | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/lib/mbedtls-2.28.0/library/bignum.c b/lib/mbedtls-2.28.0/library/bignum.c
index 62e7f76..9c256ae 100644
--- a/lib/mbedtls-2.28.0/library/bignum.c
+++ b/lib/mbedtls-2.28.0/library/bignum.c
@@ -1392,7 +1392,7 @@ void mpi_mul_hlp( size_t i,
                   mbedtls_mpi_uint *d,
                   mbedtls_mpi_uint b )
 {
-    mbedtls_mpi_uint c = 0, t = 0;
+    mbedtls_mpi_uint c = 0;
 
 #if defined(MULADDC_HUIT)
     for( ; i >= 8; i -= 8 )
@@ -1443,8 +1443,6 @@ void mpi_mul_hlp( size_t i,
     }
 #endif /* MULADDC_HUIT */
 
-    t++;
-
     while( c != 0 )
     {
         *d += c; c = ( *d < c ); d++;
-- 
2.37.1

